src/main/twirl/com/lightbend/lagom/docs/faq.scala.html

```diff
@@ -0,0 +1,57 @@
+@import com.lightbend.lagom.docs.LagomContext
+
+@(implicit context: LagomContext)
+
+@page("FAQ") {
+  <div>
+    <h1>Frequently Asked Questions</h1>
+    <blockquote>
+    Have a question about Lagom? We've sorted out a sample of frequently asked questions and provided answers below. If your question is not covered here, try these resources:
+    <ul>
+      <li><a href="//gitter.im/lagom/lagom">Lagom Gitter channel</a>
+      <li><a href="//groups.google.com/forum/#!forum/lagom-framework">lagom-framework mailing list</a> </li>
+      <li><a href="//stackoverflow.com/questions/tagged/lagom">Stackoverflow Lagom questions.</a></li>
+    </blockquote>
+
+
+      <h3>Is Lagom open source or commercial?</h3>  Lagom is
+      completely open source and has a vibrant and growing community. However,
+      you do have the option to purchase a <a href="https://www.lightbend.com/platform/subscription">Lightbend subscription</a> that
+      provides technical support as well as license to use Enterprise Suite,
+      which includes a range of powerful Application Management, Intelligent
+      Monitoring, Enterprise Integration and Advanced Tooling.     </p>
+      <h3>Does Lagom require me to use
+    Cassandra for persistence?</h2> 
+    <p> No. However, Cassandra
```

Comment:
This answer both assumes a lot of knowledge and implies some misleading things. Rather than complicating the answer further to clarify, I suggest leaving it simple, such as "No, Lagom supports using either Cassandra or relational databases, including MySQL, PostegreSQL, Oracle and H2. Cassandra provides great scalability and replication features, and is provided by default in the Lagom development environment."

I'm also not sure about the links... we want to link to more information, but we maybe don't want to assume either Java or Scala, nor do we want to have to duplicate every link. I'm wondering whether this whole page would be better off as part of the main documentation, with Java and Scala variants.